conf.yaml

```diff
@@ -1478,7 +1478,7 @@ contents:
         sections:
           -
             title:      Elastic Endpoint Security
-            prefix:     en/endpoint/guide
+            prefix:     en/endpoint
```

Comment:
Depending on how long this guide has been up, we may want to set up redirects from the current URLs to the new ones.

Reply:
Honestly, only people internally have tried to access this link so far... And there's no officially published documentation yet.

Not sure it's needed.

Reply:
Ok, then I'd agree they aren't needed 😀